File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/C/Exercicio12.c
//12. Faça um programa que receba dois números maiores que zero, calcule e mostre um elevado ao outro.

#include <stdio.h>
#include <math.h>

int main()
{
    float a, b, pot1 = 0, pot2 = 0;

    printf("Digite dois numeros a e b maiores que zero: ");
        scanf("%f %f", &a, &b);
    
    pot1 = pow(a, b);
    pot2 = pow(b, a);

    printf("a elevado a b: %.1f\n", pot1);
    printf("b elevado a a: %.1f", pot2);

    return 0;
}

File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/Java/Exercicio12.java
/*12. Faça um programa que receba dois números maiores que zero, calcule e mostre um elevado ao outro.*/

import java.util.Scanner;

public class Exercicio12 {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);

        double pot1 = 0, pot2 = 0;

        System.out.print("Digite dois números a e b maiores que zero: ");
        double a = scan.nextDouble();
        double b = scan.nextDouble();
        
        pot1 = Math.pow(a, b);
        pot2 = Math.pow(b, a);

        System.out.print("\na elevado a b: " + pot1);
        System.out.print("\nb elevado a a: " + pot2);
        System.out.print("\n");


        scan.close();
    }
    
}
